perun-core/src/test/java/cz/metacentrum/perun/core/entry/AttributesManagerEntryIntegrationTest.java

```diff
@@ -6296,6 +6338,25 @@ public void getRequiredMemberResourceAttributesFromOneServiceWorkWithUser() thro
 
 	}
 
+	@Test
+	public void getRequiredMemberResourceGroupAttributesFromOneServiceWorkWithUser() throws Exception {
+		System.out.println(CLASS_NAME + "getRequiredMemberResourceGroupAttributesFromOneServiceWorkWithUser");
+
+		vo = setUpVo();
+		member = setUpMember();
+		group = setUpGroup();
+		facility = setUpFacility();
+		resource = setUpResource();
+		service = setUpService();
+		attributes = setUpRequiredAttributes();
+		perun.getResourcesManager().assignService(sess, resource, service);
+
+		List<Attribute> reqAttr = attributesManager.getRequiredAttributes(sess, service, resource, group, member, true);
+		assertNotNull("unable to get required resource-member attributes for one service",reqAttr);
```

Comment:
I guess you can ommit "resource-member" if you do not name all other namespaces returned by the same method.